packages/theme/tools/fetch-figma-variables.ts

```diff
@@ -0,0 +1,345 @@
+import { useFigmaToDTCG } from "@tfk-samf/figma-to-dtcg"
+import type { GetLocalVariablesResponse } from "@figma/rest-api-spec"
+
+import StyleDictionary from 'style-dictionary';
+import type { Config, DesignTokens, TransformedToken } from 'style-dictionary/types';
+import { fileHeader } from 'style-dictionary/utils';
+
+import path from 'path';
+import { convertToCamelCase, convertToSnakeCase } from "./utils";
+import { ThemeOptions } from "../src";
+
+export type Organizations = 'atb' | 'fram' | 'innlandet' | 'nfk' | 'troms'
+export type Modes = 'light' | 'dark'
+export type SharedCollections = 'border' | 'spacing' | 'typography' | 'icon'
+export type OrganisationCollections = 'color_palette'
+export type VariantCollections = 'theme'
+
+const outDir = './src';
+const organizations: Organizations[] = ['atb', 'troms']
+const modes: Modes[] = ['light', 'dark'];
+
+/**
+ * Appends the name of the collection to the file path, such that
+ * it is prepended to the name of the variable (e.g., COLOR-background-neutral-...).
+ */
```

Comment:
Isn't this comment a bit misleading?
It uses token path, and the output is in camelCase

Reply:
Clarified comment